library/src/scala/tasty/reflect/Core.scala

```diff
@@ -304,6 +305,9 @@ trait Core {
     /** Pattern representing a `x: Y` type test. */
     type TypeTest = kernel.TypeTest
 
+    /** Pattern representing a `_` pattern */
+    type Wildcard = kernel.Wildcard
+
```

Comment:
A little concern for flattened `Pattern` names:  `Wildcard` has different meanings in different places, it may cause confusion.

Maybe `WildcardPattern` -- as most macros will not touch patterns, a long name will not affect usability.

Reply:
I renamed it to `WildcardPattern`